### PR TITLE
Update README for cncf project and add chinese README

### PR DESCRIPTION
**Pull request by liangyuanpeng**
Changed:

1. add describe for cncf projectr
2. add chinese README

@zheng1  @FeynmanZhou  PTAL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 ![OpenELB Logo](./doc/logo/color-horizontal.svg)
```
Comment (liangyuanpeng) [inline README.md:5]: I absolute want to update this logo, but i haven't the new logo.

Comment (FeynmanZhou) [reply README.md:5]: Let me replace it.

File: README_zh.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# OpenELB 
+
+![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/kubesphere/openelb) ![go report](https://goreportcard.com/badge/github.com/kubesphere/openelb)
+
+![OpenELB Logo](./doc/logo/color-horizontal.svg)
+
+OpenELB是一个开源的负载均衡器实现,可以在裸金属服务器、边缘以及虚拟化环境中使用LoadBalancer类型的service.目前作为[沙箱项目](https://www.cncf.io/sandbox-projects/)托管于[CNCF](https://www.cncf.io/).
```
Comment (FeynmanZhou) [inline README_zh.md:7]: 文档中有几个规范问题：
1. 中英文之间左右需各空一格；
2. 英文单词和术语首字母的大小写问题，如 Kubernetes、Service

Tip: 可以参考这个[文档](https://github.com/kubesphere/website/blob/master/localization_style_guides/KubeSphere%20Localization%20Style%20Guide%20(for%20Simplified%20Chinese).md)进行规范。

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]
Thanks for your contribution! See my comments above.

**Review by FeynmanZhou — APPROVED** [review]

**Comment by zheng1** [comment]
Thanks for your contribution! 
/approve

**Pull request merged**